pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -171,6 +172,17 @@ func (pl *VolumeZone) Filter(ctx context.Context, _ *framework.CycleState, pod *
 	return nil
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *VolumeZone) EventsToRegister() []framework.ClusterEvent {
```

Comment:
ditto, lets add a comment on each explaining how they impact schedulability of pods.

cc @cofyc